myapp/scheduler/stock_api_update.py

```diff
@@ -0,0 +1,29 @@
+from myapp.models import Stock
+from myapp import stock_api
+
+
+def update_top_stocks():
```

Comment:
the code in this method is kind of the same as the code in method `update_top_stocks` ...
recommended to centralize the code in one place and avoid duplications